stdlib/2and3/lib2to3/pgen2/tokenize.pyi

```diff
@@ -0,0 +1,30 @@
+# Stubs for lib2to3.pgen2.tokenize (Python 3.6)
+# NOTE: Only elements from __all__ are present.
```

Comment:
They by definition shouldn't be considered public if they're not in `__all__`? stubgen ignores such names for example. If `__all__` is unintentionally incomplete we might fix that in 3.7. For now though we can consider the stubs partial. I'd really like to land this sooner rather than later.